profiles-sdk-examples/docs/config.md

```diff
@@ -0,0 +1,118 @@
+# Configuration Options
+
+Cortex configuration properties can be specified in the [Spark Configurations](https://spark.apache.org/docs/latest/index.html).
+
+Configuration options are specific to your use case and may depend on:
+- How/where you intend to run your application?
```

Comment:
Remove ?s these are statements not questions even thought they start with question words